.github/workflows/apps-cypress-percy-test.yml

```
@@ -68,8 +73,10 @@ jobs:
 
       - name: Run Cypress Tests without Percy
         if: steps.test-type.outputs.skip_percy == 'true'
-        run: npm run test:cypress
+        run: npm run test:cypress || echo "Script missing, skipping this step..."
+        continue-on-error: true
```

This morning I woke up and the first thing I thought was... Will this also cause the tests to fail silently if Cypress throws an error?? To be tested 😄